### ISSUE TITLE
VGG16/19 models do not train

### ISSUE THREAD

**Issue by htrenquier** [keras-team/keras-applications#93]
I am trying to train VGG16 and VGG19 on cifar10.
However, it seems that the models are not training. The validation accuracy stagnates at 0.1 (1/10: 10 classes for the cifar10 dataset).

In the code, (line 82 for vgg19 and vgg16) it seems that the line `global backend, layers, models, keras_utils` is missing.
In comparison to other implemented models,  this line is written after the documentation/description of the definition of the model.

Keras-Applications version is 1.0.7

Thank you

**Comment by taehoonlee** [comment]
@htrenquier, I hope [the official example](https://keras.io/examples/cifar10_resnet/) will help you.

**Comment by htrenquier** [comment]
@taehoonlee thank you for answering.
However, this example is about ResNet and I don't encounter any issue when training this model.
I have written a simple example: 
https://github.com/htrenquier/ml-attribute-based-testing/blob/master/example.py
Maybe it will be easier for you to see my problem.

**Comment by taehoonlee** [comment]
@htrenquier, The convergence rate depends on many factors, such as model complexity, data distribution, and hyperparameters of an optimizer. You can try a more lighter model that VGG, or data augmentation in the official example. VGG in keras-applications has no problem.
